pilot/pkg/bootstrap/server.go

```diff
@@ -168,13 +177,23 @@ type PilotArgs struct {
 // Server contains the runtime configuration for the Pilot discovery service.
 type Server struct {
 	mesh              *meshconfig.MeshConfig
-	serviceController *aggregate.Controller
+	ServiceController *aggregate.Controller
 	configController  model.ConfigStoreCache
 	mixerSAN          []string
 	kubeClient        kubernetes.Interface
 	startFuncs        []startFunc
-	listeningAddr     net.Addr
+	HTTPListeningAddr net.Addr
+	GRPCListeningAddr net.Addr
 	clusterStore      *clusterregistry.ClusterStore
+
+	EnvoyXdsServer   *envoyv2.DiscoveryServer
```

Comment:
package private variable should declare in lowercase: such as grpcServer.

Reply:
It's intended to be public, used in the other packages.